mentors/ruby/j.json

```diff
@@ -62,5 +62,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "After years of light web development in the financial industry, I changed course to follow my passion for software development. I love working with Ruby and Javascript and am currently building my knowledge around Elixir, Java, React, and Angular."
-  }  
+  },
+  {
+  "github_username": "jubaan",
+  "name": "Julio Añoveros",
+  "link_text": null,
+  "link_url": null,
+  "avatar_url": null,
+  "bio": "Full-stack Web Developer, Ruby entusiast and avid learner. My phylosophy: 'Teaching as a way of learning'."
+  }
```

Comment:
You will want to double check the indentation.  Compare to the format of the other entries.

Reply:
Thank you @kotp.